kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml

```diff
@@ -280,32 +280,53 @@ webApi:
     table: biorxiv_medrxiv_meca_path_metadata
     dataUrl:
       urlExcludingConfigurableParameters: https://api.biorxiv.org/meca_index/elife/all
+
+  # bioRxiv/medRxiv apis
+  # bioRxiv api
   - dataset: '{ENV}'
-    table: biorxiv_medrxiv_meca_path_metadata
```

Comment:
would maybe otherwise keep those changes in a separate PR (The commit history then provides an accurate history and it is easier to review)